torch/utils/_zip.py

```diff
@@ -30,8 +30,8 @@ def remove_prefix(text, prefix):
         return text[len(prefix):]
     return text
 
-def write_to_zip(file_path, strip_file_path, zf):
-    stripped_file_path = remove_prefix(file_path, strip_file_dir + "/")
+def write_to_zip(file_path, strip_file_path, zf, prepend_str=""):
```

Comment:
It's not really a change in this PR, but why pass `strip_file_path` and then use global `strip_file_dir`?

`remove_prefix(file_path, strip_file_dir + "/")` should be changed to `remove_prefix(file_path, strip_file_path)`